packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -680,7 +680,12 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
      * @returns
      */
     $executeRaw(query: TemplateStringsArray | sqlTemplateTag.Sql, ...values: any[]) {
-      return createPrismaPromise((txId, lock, otelCtx) => {
+      return createPrismaPromise(async (txId, lock, otelCtx): Promise<unknown> => {
+        const isInstanceOfSQL = query.constructor.name === 'Sql'
+        if (isInstanceOfSQL && (query as { sql: string }).sql === '') {
+          return null
+        }
```

Comment:
Should we maybe make `isInstanceOfSQL` a type guard?

Something like

```ts
function isSql(value: unknown): value is sqlTemplateTag.Sql {
  return value.constructor.name === 'Sql'
}
```

and then, here

```ts
if (isSql(query) && query.sql === '') {
  return null
}
```

No strong opinion though, if you don't find it more readable, then  it's also fine as it is.

Reply:
Sure I added it!